### PR TITLE
Feature: Use Separate Cluster Trial Condition

### PR DESCRIPTION
**Pull request by siddharthlatest**

### CODE REVIEW

**Review by lakhansamani — APPROVED** [review]

File: src/pages/ClusterPage/ClusterPage.js [hunk 1]
```diff
@@ -170,6 +173,9 @@ export default class ClusterPage extends Component {
 				|| mark.plan.endsWith(cluster.pricing_plan)
 				|| mark.plan.startsWith(cluster.pricing_plan),
 		);
+		console.log('trial: ', isUsingClusterTrial);
+		console.log('clusters: ', cluster);
```
Comment (jyash97) [inline src/pages/ClusterPage/ClusterPage.js:177]: We need to remove this logs @ShahAnuj2610

File: src/pages/ClusterPage/ClusterPage.js [hunk 2]
```diff
@@ -250,37 +256,18 @@ export default class ClusterPage extends Component {
 
 					{cluster.status === 'active' || cluster.status === 'deployments in progress' ? (
 						<div>
-							{cluster.trial || cluster.subscription_id ? (
+							{isUsingClusterTrial || cluster.subscription_id ? (
 								<Link to={`/clusters/${cluster.id}`}>
 									<Button type="primary">View Details</Button>
 								</Link>
-							) : null}
-							{cluster.trial || cluster.subscription_id ? (
```
Comment (jyash97) [inline src/pages/ClusterPage/ClusterPage.js:258]: Did you confirm this change ? I guess this was used to fire Payment Modal based on URL.

Comment (ShahAnuj2610) [reply src/pages/ClusterPage/ClusterPage.js:258]: these change was done by @siddharthlatest. He would best know it.

File: src/pages/ClusterPage/ClusterPage.js (hunk 1 above)
Comment (ShahAnuj2610) [reply src/pages/ClusterPage/ClusterPage.js:177]: these logs were put by @siddharthlatest. He would best know it.

File: src/pages/ClusterPage/ClusterPage.js (hunk 2 above)
Comment (siddharthlatest) [reply src/pages/ClusterPage/ClusterPage.js:258]: We should remove them. We needed this deployed and there was debugging needed, but I missed removing them prior to commiting.

**Pull request merged**